profile.php

```diff
@@ -17,14 +17,14 @@
 			break;
 		case "login":
 			$_SESSION["user-id"] = '1';
-			$_SESSION["user-level"] = '1';
+			// $_SESSION["user-level"] = '1';
```

Comment:
$_SESSION["user-level"] remove korar drkar nei kuthao ei.... accInfo table e primary key amra customer_id+customer_level set korechilam.... So 2tai lagbe amader and query e soraleo prob nei thakleo prob nei... DB design flaw.... But amader age sikhai ni...last week er pora ei gula... Ekhn jehetu primary key 2tai diye felechi I don't think DB change na kore eita remove korar dorkar ache... and ei stage e DB change o kora possible na...